datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml

```diff
@@ -35,100 +35,88 @@ dag:
-          SOURCE_FILE: "files/data.csv"
-          TARGET_FILE: "files/data_output.csv"
+          PIPELINE_NAME: "{{ var.json.covid19_google_mobility.mobility_report.pipeline_name }}"
+          SOURCE_URL: "{{ var.json.covid19_google_mobility.mobility_report.source_url }}"
+          SOURCE_FILE: "{{ var.json.covid19_google_mobility.mobility_report.source_file }}"
+          TARGET_FILE: "{{ var.json.covid19_google_mobility.mobility_report.target_file }}"
+          CHUNKSIZE: "{{ var.json.covid19_google_mobility.mobility_report.chunksize }}"
+          PROJECT_ID: "{{ var.value.gcp_project }}"
+          DATASET_ID: "{{ var.json.covid19_google_mobility.mobility_report.dataset_id }}"
+          TABLE_ID: "{{ var.json.covid19_google_mobility.mobility_report.table_id }}"
           TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
-          TARGET_GCS_PATH: "data/covid19_google_mobility/mobility_report/data_output.csv"
-          PIPELINE_NAME: "mobility_report"
-          CSV_HEADERS: >-
-            ["country_region_code" ,"country_region" ,"sub_region_1" ,"sub_region_2" ,"metro_area" ,"iso_3166_2_code" ,"census_fips_code" ,"place_id" ,"date" ,"retail_and_recreation_percent_change_from_baseline" ,"grocery_and_pharmacy_percent_change_from_baseline" ,"parks_percent_change_from_baseline" ,"transit_stations_percent_change_from_baseline" ,"workplaces_percent_change_from_baseline" ,"residential_percent_change_from_baseline"]
-          RENAME_MAPPINGS: >-
-            {"country_region_code":"country_region_code" ,"country_region":"country_region" ,"sub_region_1":"sub_region_1" ,"sub_region_2":"sub_region_2" ,"metro_area":"metro_area" ,"iso_3166_2_code":"iso_3166_2_code" ,"census_fips_code":"census_fips_code" ,"place_id":"place_id" ,"date":"date" ,"retail_and_recreation_percent_change_from_baseline":"retail_and_recreation_percent_change_from_baseline" ,"grocery_and_pharmacy_percent_change_from_baseline":"grocery_and_pharmacy_percent_change_from_baseline" ,"parks_percent_change_from_baseline":"parks_percent_change_from_baseline" ,"transit_stations_percent_change_from_baseline":"transit_stations_percent_change_from_baseline" ,"workplaces_percent_change_from_baseline":"workplaces_percent_change_from_baseline" ,"residential_percent_change_from_baseline":"residential_percent_change_from_baseline"}
+          TARGET_GCS_PATH: "{{ var.json.covid19_google_mobility.mobility_report.target_gcs_path }}"
+          SCHEMA_PATH: "{{ var.json.covid19_google_mobility.mobility_report.schema_path }}"
+          DROP_DEST_TABLE: "{{ var.json.covid19_google_mobility.mobility_report.drop_dest_table }}"
+          INPUT_FIELD_DELIMITER: "{{ var.json.covid19_google_mobility.mobility_report.input_field_delimiter }}"
+          REMOVE_SOURCE_FILE: "{{ var.json.covid19_google_mobility.mobility_report.remove_source_file }}"
+          DELETE_TARGET_FILE: "{{ var.json.covid19_google_mobility.mobility_report.delete_target_file }}"
```

Comment:
We can expose these variables (except `TARGET_GCS_BUCKET`) so it's clear to readers of the code what values these variables should have

Reply:
Done.